installation-and-configuration/configure-winrm-authentication.md

```diff
@@ -0,0 +1,35 @@
+---
+description: >-
+  This article explains how to configure WinRm authentication on
+  your machine to successfully run snapshots.
+---
+
+# Configure Windows Remote Management (WinRM)
+
+In order to successfully collect data when creating a snapshot, **SysKit Trace** will use a **remote PowerShell session** to connect to Office 365. This is required to collect the data for Security & Compliance, Exchange Online and some Teams reports.
+
+To make sure that collection can work properly, Windows Remote Management (WinRM) on your computer needs to allow **Basic authentication** (it's enabled by default).
+
+**Multi - Factor Authentication**
```

Comment:
i think Multi - Factor Authentication should be without spaces, at least it's like that in the Microsoft docs.